keyboards/omron75/config.h

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2019 Maarten Dekkers <maartenwut@gmail.com>
```

Comment:
are you maartenwut?

Reply:
:) Nope. Although he did the pcb/QMK work for the GB runner. I am merely helping the GB runner merge the QMK part. I’ll work through rest of the comments in a bit.

Reply:
:) Nope. Although he did the pcb/QMK work for the GB runner. I am merely helping the GB runner merge the QMK part. I’ll work through rest of the comments in a bit.